nexus-config/src/nexus_config.rs

```diff
@@ -303,6 +304,11 @@ pub struct Tunables {
     ///
     /// If "None", nexus loops forever during initialization.
     pub load_timeout: Option<std::time::Duration>,
+
+    /// The amount of disk space to reserve for non-Crucible / control plane
+    /// storage in gibibytes. This amount represents a buffer that the region
+    /// allocation query will not use for each U2.
+    pub control_plane_storage_buffer_gb: u32,
```

Comment:
If I'm reading this right: this is part of `PackageConfig`, which is documented as "configuration parameters known at compile-time"; as opposed to `DeploymentConfig`, which are runtime, deployment-specific tunables.  But isn't the point of doing it this way to make it runtime?

Edit: I think this is basically fine -- see my long comment below for more.